tests/cypress/template/global-search.cy.js

```diff
@@ -1,3 +1,57 @@
 describe('Global search page', () => {
-  it('no-op test', () => {})
+  const term = 'bass';
+  it('Contains title and title metadata', () => {
+      cy.visit(`/en/search?clang=en&q=${term}`)
+
+      const expectedTitle = "'bass' - Search results - Skosmos being tested"
```

Comment:
This of course needs to be changed as well if you change the title in the template above, as suggested.